assets/javascripts/discourse/templates/modal/create-channel-modal.hbs

```diff
@@ -0,0 +1,92 @@
+{{#d-modal-body title="chat.create_channel.title"}}
```

Comment:
`create-channel` please, otherwise it will generate `-modal-modal` as class name

Reply:
You probably want to double check this, I think there's been multiple issues in the merge